CMakeLists.txt

```diff
@@ -192,8 +198,12 @@ else()
 endif()
 
 if(NOT WIN32)
-  core_optional_dep(OpenGl)
-  if(OPENGL_FOUND AND NOT APPLE)
+  if(NOT ROCKCHIP_FOUND)
+    core_optional_dep(OpenGl)
+  else()
+    core_optional_dep(XRandR ENABLE_X11)
+  endif()
+  if(OPENGL_FOUND AND NOT APPLE AND NOT ROCKCHIP_FOUND)
```

Comment:
I don't think the already existing nonsense should be extended here. even a blind man can see that this leads to nowhere